packages/ui/src/primitives/tailwind/Select/index.tsx

```diff
@@ -155,17 +158,21 @@ const Select = <T extends OptionValueType>({
         onClick={toggleDropdown}
         onMouseDown={handleMouseDown}
         endComponent={
-          <MdOutlineKeyboardArrowDown
-            size="1.5em"
-            className={`mr-2 transition-transform ${showOptions.value ? 'rotate-180' : ''} ${
-              disabled ? 'opacity-50' : ''
-            }`}
-            onClick={() => {
-              if (!disabled) {
-                toggleDropdown()
-              }
-            }}
-          />
+          loading ? (
+            <LoadingView spinnerOnly className="mr-2 h-6 w-6" />
```

Comment:
`LoadingView` here with a fixed set of props might affect the component's dynamic nature and is tightly coupled. For example, if we need to provide text to the `LoadingView` we will need to add another prop in `SelectProps` which increases prop complexity.